src/scaled-model.jl

```diff
@@ -0,0 +1,236 @@
+export ScaledModel
+
+struct IpoptScaling{T}
```

Comment:
Is this really specific to Ipopt?

Reply:
I renamed the scaling `ConservativeScaling`. This is not specific to Ipopt, others solvers are using this scaling as well.